relay-general/src/store/normalize.rs

```diff
@@ -749,7 +750,9 @@ pub fn light_normalize_event(
             config.max_secs_in_future,
         )?; // Timestamps are core in the metrics extraction
         normalize_event_tags(event)?; // Tags are added to every metric
-        normalize_device_class(event);
+        if config.device_class_synthesis_config {
+            normalize_device_class(event);
+        }
```

Comment:
I assume this PR is related to https://github.com/getsentry/relay/pull/1970. This will only normalize `device.class` when the feature flag is enabled for the project, so the functionality your other PR does won't run for organizations with the feature flag disabled. It will also skip other functionality, so no `device.class` tags will exist on events. Is this what you want?

Reply:
Hi @iker-barriocanal , yup this is the effect we're looking for. We're starting to rollout device class to customers and we want to be able to turn off this functionality in case the classifications are wrong or something breaks.